docs/group_guidelines/git_standards.md

```diff
@@ -0,0 +1,86 @@
+# Git - Standard Practices
+The purpose of this document is to provide our team with a set of standards to follow throughout our capstone project.
+With this document, we hope to ensure that we stay organized and consistent throughout the term.
+
+# Branches
+
+We will be following a technique called feature branches, where we create a new branch for every feature which is to be
+developed. This allows us to prevent merge conflicts, and closely monitor each new feature we need to create. As well,
+feature branches ensure we are dividing properly dividing issues into achievable tasks. Once the feature has been 
+completed, and the branch has been merged to main, we will NOT be deleting the branch. If bugs occur with the feature, 
+we can return to the branch in order to fix the bug. 
+
+## Naming:
+Branch names should be consistent throughout our repository. Below is the naming scheme which we should follow for all 
+feature branches: 
+
+<issue-id>_<brief-feature-description>
+
+Ex: 7_create-git-standards-documentation
+
+This naming scheme ensures that we are properly creating issues for each branch in order to ensure gitstats stays happy
+
+### Dashes & Underscores
+To stay consistent with the way GitHub displays spaces in words, please use - in your branch name for spaces, and _ for
+the separator between the issue id and the feature description. 
+
+# Pull Requests
+
+## Creation:
+PR names and branch names will follow a similar syntax, as everything is based on issues.
+### Title:
+Pull request names should follow a similar syntax to branch names. They should start with the issue # which they aim to 
+close, and then continue on to provide a short informative summary of what the pull request accomplishes. Below is the
+naming scheme we should follow for all pull requests: 
+<issue-id> <short PR summary>
+### Description:
+Above all else, please ensure you use the closing keyword within your PR description, this ensures that the issue tagged 
+is successfully resolved. If your PR does not close an issue, explain why. Your description should explain what you did,
+why you did it, and if there are any known issues please mention them within the pull request.
+
+## Review:
+The goal when reviewing a pull request is to ensure that all code follows the standards we have defined within our code
+standards document - to be created when our tech stack is defined.
+
+When leaving comments, be sure you are providing constructive criticism, and if you request changes to the pull request,
+please describe precisely what requires change, and why it requires change.
+
+## Merging:
+As a rule of thumb, it is the responsibility of the author of the pull request to merge the code once it has been 
+approved.
+
+# Issues:
+
```

Comment:
heads up: inconsistent spacing for headings in this document

Reply:
Title & description are sections under creation, then review and merging are two separate categories - let me know if you want this changed!

Reply:
I mean having the line spacing between the headings and the paragraphs, but the markdown preview probably looks the same. It's just for future reports to be aware!